appserver/payara-appserver-modules/microprofile/healthcheck/src/main/java/fish/payara/microprofile/healthcheck/HealthCheckService.java

```diff
@@ -494,5 +502,25 @@ public UnprocessedChangeEvents changed(PropertyChangeEvent[] events) {
 
         return new UnprocessedChangeEvents(unchangedList);
     }
-
+    
+      /**
+     * Gets the application name from the invocation manager.
+     *
+     * @return The application name
+     */
+    public String getApplicationName() {
```

Comment:
As far as I can see from the changes this method is never used in the service class but only called from the outside and it does not use any of the service specific state. It should not be here but where it is actually used or as it basically is a static method it could just as well be in a utility so it can be reused easier. I am pretty sure this exact method already exists multiple times in other places where the same issue exists. Would be nice if we finally make it a helper instead of duplicating the code again.